packages/rancher-istio-1.8/charts/README.md

```diff
@@ -51,12 +51,18 @@ http://{{ .Values.nameOverride }}-grafana.{{ .Values.namespaceOverride }}.svc:{{
 The url depends on the default values for `nameOverride`, `namespaceOverride`, and `grafana.service.port` being set in your rancher-monitoring or other monitoring instance.
 
 ##### Tracing
-The `kiali.external_services.tracing` url and `.Values.tracing.contextPath` is set in the rancher-istio values.yaml:
+The `kiali.external_services.tracing.url` url, `kiali.external_services.tracing.in_cluster_url` url and `.Values.tracing.contextPath` is set in the rancher-istio values.yaml. To ensure traces work for your services in the Kiali dashboard, you must set `kiali.external_services.tracing.url` and `kiali.external_services.tracing.in_cluster_url` key value using the complete external url to your tracing instance.
```

Comment:
nit: Should we split sentences into newlines?  Also, what do you think about removing "url" after the url values since contextPath does not have a proceeding name?

Reply:
These should be fixed in the most recent commit